modules/ROOT/pages/deployment/container/container-setup.adoc

```diff
@@ -208,30 +317,15 @@ By default, containers created with _docker run_ are given a random name like `s
 See the details in the _docker run_ documentation for available options. Consider `always` as a good starting point.
 
 --mount: Attach a filesystem mount to the container::
-{docker-mount-url}[Docker volumes] are completely managed by Docker and have no server OS dependency. See {docker-mount-nfs-url}[Create a service which creates an NFS volume] for an example. Note the volume mount target path `target=/var/lib/ocis` which uses the default oCIS data path if not otherwise defined.
+{docker-mount-url}[Docker volumes] are completely managed by Docker and have no server OS dependency. See {docker-mount-nfs-url}[Create a service which creates an NFS volume] for an example. Note the volume mount target path `target=/var/lib/ocis` which uses the default oCIS data path if not otherwise defined. Note that the directory on the host must already exist, it will not be created. 
 
 --volume, -v: Bind mount a volume::
-{docker-bindmount-url}[Bind mounts] depend on the directory structure and OS of your server. Use this type to mount a local directory of your OS. Example: `-v /some/host/dir:/var/lib/ocis` which uses the default oCIS data path if not otherwise defined.
+{docker-bindmount-url}[Bind mounts] depend on the directory structure and OS of your server. Use this type to mount a local directory of your OS. Example: `-v /some/host/dir:/var/lib/ocis` which uses the default oCIS data path if not otherwise defined. If the bind-mount points to a directory that does not yet exist on the host, the directory will be created automatically - using the user the docker _service_ runs with, usually the root user.
```

Comment:
```suggestion
{docker-bindmount-url}[Bind mounts] depend on the directory structure and OS of your server. Use this type to mount a local directory of your OS. Example: `-v /some/host/dir:/var/lib/ocis`, which uses the default oCIS data path if not otherwise defined. If the bind-mount points to a directory that does not yet exist on the host, the directory will be created automatically using the user the docker _service_ runs with, usually the root user.
```